tests/periph_wdg/main.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2019 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test for wdg Drivers
+ *
+ *              This test initializes the wdg counter to a preset value
+ *              and spins until a wdg reset is triggered
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "periph/wdg.h"
+#include "timex.h"
+
+#ifndef WDG_RESET_TIME
+#define WDG_RESET_TIME      (2U*US_PER_SEC)
```

Comment:
I would keep 2U here, but move the multiplication with `US_PER_SEC` when calling `wdg_init`. Otherwise, you get this message when the board is booting:
`Application should reset every ~ 2000000 seconds` instead of `Application should reset every ~2 seconds`